tests/frontends/torch/test_torch_norm.py

```diff
@@ -18,14 +18,21 @@
 @pytest.mark.parametrize('normalized_shape', [2])
 @pytest.mark.parametrize('dtype', [torch.float32])
 def test_layer_norm(shape, normalized_shape, dtype):
-    check_module(torch.nn.LayerNorm(normalized_shape=normalized_shape), [torch.randn(shape, dtype=dtype)])
+    check_module(
+        torch.nn.LayerNorm(normalized_shape=normalized_shape), [torch.randn(shape, dtype=dtype)], atol=1e-4, rtol=1e-4
+    )
 
 
 @pytest.mark.parametrize('shape', [[1, 4, 32, 32]])
 @pytest.mark.parametrize('num_groups', [1, 2, 4])
 @pytest.mark.parametrize('dtype', [torch.float32])
 def test_group_norm(shape, num_groups, dtype):
-    check_module(torch.nn.GroupNorm(num_groups=num_groups, num_channels=shape[1]), [torch.randn(shape, dtype=dtype)])
+    check_module(
+        torch.nn.GroupNorm(num_groups=num_groups, num_channels=shape[1]),
+        [torch.randn(shape, dtype=dtype)],
+        atol=1e-2,
+        rtol=1e-2,
```

Comment:
Does current implementation pass the atol=rtol=1e-4 precision?

Reply:
removed